src/openforms/prefill/contrib/objects_api/plugin.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from typing import Any, Iterable
+
+from django.utils.translation import gettext_lazy as _
+
+from openforms.authentication.service import AuthAttribute
+from openforms.submissions.models import Submission
+from openforms.typing import JSONEncodable
+
+from ...base import BasePlugin
+from ...constants import IdentifierRoles
+from ...registry import register
+
+logger = logging.getLogger(__name__)
+
+PLUGIN_IDENTIFIER = "objects_api"
+
+
+@register(PLUGIN_IDENTIFIER)
+class ObjectsAPIPrefill(BasePlugin):
+    verbose_name = _("Objects API")
+    requires_auth = AuthAttribute.bsn
```

Comment:
I see this being used for both BSN/KVK authentication methods. Maybe requiring auth at the plugin level is too much, but instead when the prefill runs, we can decide at runtime which auth (BSN or KVK) should be used and validated.

```suggestion
```